lib/utils.tsx

```diff
@@ -144,8 +144,8 @@ const utils = {
       })
     }
   },
-  toastSuccess: (message: string, showDialog?: boolean) => {
-    if (message.length < 100 && !showDialog) {
+  toastSuccess: (message: string, type?: 'toast' | 'dialog') => {
+    if ((type !== 'dialog' && message.length < 100) || type === 'toast') {
```

Comment:
let's add them (`toast`, `dialog`) to a const?
(and maybe to make it able to import this const from components by other projects?)